src/api/config/routes.rb

```diff
@@ -731,10 +731,10 @@ def self.public_or_about_path?(request)
   end
 
   # StagingWorkflow API
-  scope module: 'staging' do
-    resources :staging_projects, only: [:show], param: :name do
-      get 'staged_requests' => 'staged_requests#index', constraints: cons
-      resource :staged_requests, controller: 'staged_requests', only: [:create, :destroy], constraints: cons
+  resources :staging, only: [], param: 'main_project_name' do
+    resources :staging_projects, only: [:index, :show], controller: 'staging/staging_projects', param: :name do
+      get 'staged_requests' => 'staging/staged_requests#index', constraints: cons
```

Comment:
This shouldn't be part of `resources :staged_requests`?

Reply:
There is no resources but resource, with resources you don't have an index action. Changing to resources is not easily possible because it would create a delete route like this:

```
DELETE /staged_requests/:id
```

However, in the API we provide the id in the body. That's the reason why we use resource + additional index action.